research/src/pages/select/select.research.behavior.mdx

```diff
@@ -263,53 +263,26 @@ Currently this is testing the built-in browser based `<select>` control
   <option value="ZW">Zimbabwe</option>
 </select>
 
-## States:
+## Pressing escape
 
-**Open** - this is when the popup is open and the state is placed on the select.
+While adding in the events I noticed an interesting user experience that I
```

Comment:
With the above spec, where arrow up/down *sets focus but does not select* I'd expect escape to revert to the previous selection, because no selection operation occurred prior to escaping the modal.

FWIW [ARIA](https://www.w3.org/TR/wai-aria-practices/#listbox_kbd_interaction) describes the selection behavior as optional in single-select listboxes.

Reply:
> FWIW ARIA describes the selection behavior as optional in single-select listboxes.

And I'd like to make it so that it isn't optional, I'll add this to the issue and will probably loop in the aria folks.

Reply:
Issue is here: https://github.com/WICG/open-ui/issues/102